src/main/java/uk/gov/cslearning/record/api/InviteController.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.cslearning.record.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.cslearning.record.domain.Event;
+import uk.gov.cslearning.record.domain.Invite;
+import uk.gov.cslearning.record.repository.EventRepository;
+import uk.gov.cslearning.record.repository.InviteRepository;
+
+import java.util.Collection;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+@RestController
+@RequestMapping("/event")
+public class InviteController {
+
+    @Autowired
+    EventRepository eventRepository;
+
+    @Autowired
+    InviteRepository inviteRepository;
+
+    public InviteController(InviteRepository inviteRepository, EventRepository eventRepository){
+        checkArgument(inviteRepository != null);
+        checkArgument(eventRepository != null);
+        this.inviteRepository = inviteRepository;
+        this.eventRepository = eventRepository;
+    }
+
+    @GetMapping("/{eventId}/invitee")
+    public ResponseEntity<Collection<Invite>> listInvitees(@PathVariable("eventId") String eventUid){
+        Collection<Invite> result = inviteRepository.findByEventId(eventUid);
```

Comment:
Can you create an InviteService and call the repository from there instead of calling the repository directly from the controller. The service methods should be transactions.